Fix docs staging and production builds

Staging site has been broken since UTC Wed Feb 22 11:00:02 AM.

Local build throws this error:

```
Error: Markdown link URL is mandatory in "eosdocs/DUNE/index.md" file (title: https://github.com/kislyuk/argcomplete#global-completion, line: 647).
[ERROR] Client bundle compiled with errors therefore further build is impossible.
FATAL: npm run build failed exiting
```

--- a/scripts/clean_rebuild.sh
+++ b/scripts/clean_rebuild.sh
@@ -146,6 +146,9 @@ do
   if [ "${gitrepo}" == "eosnetworkfoundation/eos-system-contracts" ]; then
     branch="v3.1.1"
   fi
+  if [ "${gitrepo}" == "AntelopeIO/DUNE" ]; then
+    branch="v1.1.0"
+  fi
 
   if [ -z "$branch" ]; then
     "${SCRIPT_DIR:?}"/generate_documents.sh -d "$ARG_BUILD_DIR" -r ${gitrepo} "$CMD_FLAGS"

--- a/scripts/clean_rebuild.sh
+++ b/scripts/clean_rebuild.sh
@@ -122,6 +122,8 @@ if [ -n "$ARG_STAGING" ]; then
   CMD_FLAGS="-xs"
 fi
 
+dune_branch="main"
+
 for gitrepo in eosnetworkfoundation/docs \
     AntelopeIO/cdt \
     eosnetworkfoundation/eos-system-contracts \
@@ -148,6 +150,7 @@ do
   fi
   if [ "${gitrepo}" == "AntelopeIO/DUNE" ]; then
     branch="v1.1.0"
+    dune_branch=${branch}
   fi
 
   if [ -z "$branch" ]; then
@@ -188,7 +191,7 @@ set -x
 # USE DUNE because it is a one file change and its fast
 # note generate documents can take multiple hosts args -h u@host1 -h u@host2 ...
 if [ -n "$ARG_HOST" ] && [ -n "${ARG_IDENTITY}" ]; then
-  "${SCRIPT_DIR:?}"/generate_documents.sh -d "$ARG_BUILD_DIR" -f -r "AntelopeIO/DUNE" -h "$ARG_HOST" -i "$ARG_IDENTITY" -c "${ARG_CONTENT_DIR:-~/content}" "$CMD_FLAGS"
+  "${SCRIPT_DIR:?}"/generate_documents.sh -d "$ARG_BUILD_DIR" -f -r "AntelopeIO/DUNE" -b "$dune_branch" -h "$ARG_HOST" -i "$ARG_IDENTITY" -c "${ARG_CONTENT_DIR:-~/content}" "$CMD_FLAGS"
 fi
 
 ## All done, remove the lock file, and set last updated times

